### ISSUE TITLE
Refine the JupyterHub "landing page" to standardize our messaging

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/team-compass#444]
We've had team discussion about updating and streamlining the project's impact statement and value proposition at a product level, we can use this as an opportunity to ensure that the landing pages above serve the following goals:

### To do

- [x] Create jupyterhub.github.io and point it to hub.jupyter.org
- [ ] Decide on the goals / target audiences we need to meet with our project docs
- [ ] Decide which locations on the web are the ideal space for each audience
- [ ] ...decide an implementation plan (including deleting some resources if we wish)

### List of sort-of landing pages

We have a few different landing-page spaces for JupyterHub, some with inconsistent information:

- https://jupyter.org/hub <-- technical source of truth but hasn't been updated by us in ages
- https://hub.jupyter.org <-- very incomplete re-working we did a few years ago
- https://compass.hub.jupyter.org <-- project-specific docs but is meant more for our team than the outside world
- https://jupyterhub.readthedocs.io <-- product docs for jupyterhub but can be confused w/ project-wide docs

### Implementation suggestion from @jasongrout

Define a vision for each web resource, like:

1. the audience
2. the questions it addresses (i.e., why someone came to the page)
3. the goals for the page (i.e., what we expect the page to accomplish) and ways to measure those goals

I imagine for [jupyter.org/hub](http://jupyter.org/hub), for example:

1. The audience is the general public, perhaps (a) people who have read the small JupyterHub section on [jupyter.org](http://jupyter.org/) and clicked through to this page, or (b) searched google for "JupyterHub", or (c) had a friend that tried to explain JupyterHub in like 10 seconds, and sent this link for more info
2. who clicked on the page with questions like, "I've never heard of JupyterHub, what is it?", "Why do I care about JupyterHub", "Do I want to know more?"
3. with a goal of:
  a. giving the public an idea of where JupyterHub fits in to Project Jupyter, measured by qualititative user surveys or a short thumbs up/down question at the end of the page, or even just looking at the average scroll depth of a visitor on the page)
  b. interested people clicking through to more detailed community docs (perhaps measured by click-throughs)

With that vision, for example, the page should probably be short (though longer than the JupyterHub section on [jupyter.org](http://jupyter.org/)), high-level and approachable, with links through to more complete documentation

**Comment by sgibson91** [comment]
I'd like to breathe life into this issue!

Running the Outreachy contribution period in Nov 22, it became _very_ apparent, _very_ quickly that JupyterHub does not have a clear pathway to explain (at a high level) what the project is about, how to get around, etc etc. Pieces of this information are dotted around our docs, but not centralised in a single place that is easy to find and easy to follow, particularly for folk who have never heard of a JupyterHub before.

Chris has some great ideas for implementation above, which we can continue to discuss, and I'd like to provide some ideas for _content_ of this landing page.

- What is Project JupyterHub about?
- How does JupyterHub differ from Jupyter Notebooks, or just the term "Jupyter" (we should be brief with this as this is not really "our" problem to solve, but symptomatic of a lack of clarity from the Project Jupyter level)
- (Optional) How is JupyterHub as a project different from those that might be backed by a company?
  - This is optional as I believe it is an Outreachy-specific question when an applicant may be choosing between JupyterHub and, say, Mozilla to work with
  - I think the crux of this is _how can you expect the team dynamics to be different?_ E.g. JupyterHub is a globally distributed base of primarily volunteers, so how we work together is of a different style and a different pace to projects that have full-time paid devs and/or a headquartered office.
- What skills are required to contribute to the project? (python packaging, kubernetes, sysadmin, etc)
- What skills are we looking to _add_ to the team? (e.g. react, accessibility, etc)
- How to navigate around the organisation
- ... etc. please add if you think of anything else

As I mentioned before, we already have parts of this written up, they're just in distributed places. I also think we should try to revive the excellent work that was started in this PR as part of this! https://github.com/jupyterhub/jupyterhub/pull/2726

**Comment by sgibson91** [comment]
It would be really cool if we could embed the demo video into this page as well, now that it's published! https://www.youtube.com/watch?v=IhMHMuwphc0

**Comment by choldgraf** [comment]
I suggest at least to start we take these steps:

1. Create a jupyterhub.github.io repository that we can use for the jupyterhub org docs.
2. Ask whoever controls jupyter.org to register hub.jupyter.org and point to the GitHub DNS servers
3. Configure GH pages to serve to hub.jupyter.org

Then at least we only need to update our repo to make changes and don't need to wait for DNS etc

**Comment by sgibson91** [comment]
@choldgraf are you able to lead on one or more of those action points?

**Comment by choldgraf** [comment]
I can give it a shot and see how far I get this evening or weekend 👍

**Comment by choldgraf** [comment]
### Update: a skeleton site is live

Hey all, I've taken these actions:

- I created https://github.com/jupyterhub/jupyterhub.github.io
- I've added a skeleton Sphinx site with the jupyterhub-sphinx-theme
- I've moved the jupyterhub-sphinx-theme to our organization (https://github.com/jupyterhub/jupyterhub-sphinx-theme)
- I've updated the DNS entries for jupyter.org with a `CNAME` so that hub.jupyter.org points to jupyterhub.github.io

### Still to do (by anybody, not just me!)

Add content, layout, etc to the Sphinx site that is at jupyterhub.github.io. This will then update hub.jupyter.org via a GitHub Action that builds the site.

**Comment by sgibson91** [comment]
Thank you @choldgraf!

**Comment by consideRatio** [comment]
> I've updated the DNS entries for jupyter.org with a CNAME so that hub.jupyter.org points to jupyterhub.github.io

This had some consequences for anything referencing jupyterhub.github.io that doesn't support being redirected. Thankfully for example `helm install` did so, allowing helm install commands to keep functioning.

In z2jh and binderhub's CI system we had `curl -sS https://jupyterhub.github.io/helm-chart/info.json` which didn't include the curl `-L` flag to follow redirects, so that broke.

**Comment by consideRatio** [comment]
Since for example `helm install` can handle a reference to an URL that redirects by following the redirect, we are okay I think. I think the only thing that has broken is our own and other peoples `curl` scripts etc that may not be configured as relaxed as typical browsers or even `helm` as a CLI that does follow redirects.

Let's go with it at least until we learn about something bigger failing.

**Comment by choldgraf** [comment]
Makes sense - I guess one point to consider is that if people are relying on a specific address for stuff we might want it to be something that we control rather than a GitHub-generated one? But I don't feel super strongly about that

**Comment by choldgraf** [comment]
### Blog list PR

Here's a PR that will pull the latest list of blog posts tagged `jupyterhub` on the jupyter blog!

- https://github.com/jupyterhub/jupyterhub.github.io/pull/1
